pkg/unvanquished_src.dpkdir/ui/transition_loading.rml

```diff
@@ -1,13 +1,8 @@
 <rml>
 	<head>
 		<link type="text/rcss" href="/ui/shared/basics.rcss" />
+		<link type="text/rcss" href="loading.rcss" />
```

Comment:
I don't find such a file in the tree

Reply:
Ooopsie right, I forgot the `git add`. 🤪️

Now fixed.